source/features/infinite-commit-list-scroll.tsx

```diff
@@ -0,0 +1,92 @@
+import select from 'select-dom';
+import React from 'dom-chef';
+import fetchDom from '../libs/fetch-dom';
+import features from '../libs/features';
+
+// Used feature: infinite-scroll as template
+
+// Button from: https://primer.style/css/components/loaders
+const githubLoadingButton = <button className="btn mn-3" disabled><span>Loading</span><span className="AnimatedEllipsis"></span></button>;
+
+let container: HTMLElement;
+let link: HTMLAnchorElement | undefined;
+
+let lastDate: string;
+
+function getLastDate(document_?: HTMLDivElement): string {
+	if (!document_) {
+		document_ = select<HTMLDivElement>('.commits-listing')!;
```

Comment:
Are you sure you need this? It think it should just default to `document`

Reply:
Defaulting to `document` gives an 
> Type 'HTMLElement' is missing the following properties from type 'Document': URL, activeElement, alinkColor, all, and 86 more.

And I can't change `_document` to type `Document` because what fetchDom returns `Element`